### ISSUE TITLE
Add apko?

### ISSUE THREAD

**Issue by znewman01** [nishakm/distro-tools-for-containers#1]
https://github.com/chainguard-dev/apko (disclaimer: a project of my employer, though it's OSS under Apache 2.0) seems like a good fit for this project.

There's also a related tool, https://github.com/chainguard-dev/melange, for packaging APKs that can make it into your final image via apko.

**Comment by joshuagl** [comment]
Yeah, this would be great. I started capturing some notes on apko/melange on my dev fork (https://github.com/joshuagl/distro-tools-for-containers/commit/1435069f986a4029f4dfe8ccb17761313a4bb538) but haven't gotten around to actually playing with the tools yet :-|
